dist/index.html

```diff
@@ -24,19 +24,22 @@ <h1 class="title">Today's To Do</h1>
 
     <div class="input-section">
       <label for="newtask">
-        <input type="text" id="newtask" name="newtask"
+        <input type="text" id="new-task" name="newtask"
           value="" placeholder="Add to your list...">
       </label>
-      <button type="button" class="icon">
+      <button type="button" class="icon"
+        id="add-task-button">
```

Comment:
- [ ] OPTIONAL: This button ended up in a rather weird place on your app.  I highly recommend you fix it and put it on it's correct place:

![image](https://user-images.githubusercontent.com/57421823/154447261-5c243547-f95d-44bc-b018-b69859b9bf8e.png)

That way, you will improve the overall look and feel of your app. 💪

Reply:
Hi @Meltrust :wave: 
Thanks for the approval
Can you list the steps you did to get the weird location for the **arrow button**?
Because It's working for me!
I made a [Loom](https://www.loom.com/share/9156d14368534c5ca450e75dc4396e11) video
Thanks,

Reply:
Found the issue, and the solution:

Firefox needs to know explicitly that top: 0 (tested again in Chrome with the new rule property and still works, you can safely add it to that button element).

![image](https://user-images.githubusercontent.com/57421823/154458831-a6394f53-1fb5-4a3c-ae7f-0eb0cfc8f4a8.png)

That way, it will work well in both browsers

Reply:
Hi @Meltrust,
Ooh good one!!
I would never have thought of checking on other browsers.
As you said, I use Chrome.
Thanks, You are really good at code reviews :clap:
And finding bugs :smile: